lib/encoder.py

```diff
@@ -813,9 +815,15 @@ def ReadEncodingResult(self, encoding, scoredir=None):
       with open(filename, 'r') as resultfile:
```

Comment:
I think I'll leave it like this. Filed #25 to remember to remove it when AST repos are gone.